sample-android-app/src/main/java/com/badoo/reaktive/sample/android/MainActivity.kt

```diff
@@ -1,37 +1,62 @@
 package com.badoo.reaktive.sample.android
 
+import android.annotation.SuppressLint
 import android.os.Bundle
 import android.support.v7.app.AppCompatActivity
 import android.widget.Button
 import android.widget.TextView
+import com.badoo.reaktive.disposable.CompositeDisposable
+import com.badoo.reaktive.observable.map
 import com.badoo.reaktive.observable.observableFromFunction
 import com.badoo.reaktive.observable.observeOn
+import com.badoo.reaktive.observable.onErrorReturn
 import com.badoo.reaktive.observable.startWithValue
 import com.badoo.reaktive.observable.subscribe
 import com.badoo.reaktive.observable.subscribeOn
-import com.badoo.reaktive.scheduler.ioScheduler
+import com.badoo.reaktive.observable.switchMap
+import com.badoo.reaktive.scheduler.computationScheduler
 import com.badoo.reaktive.scheduler.mainScheduler
+import com.badoo.reaktive.subject.publish.publishSubject
 import java.text.SimpleDateFormat
-import java.util.Date
+import java.util.*
```

Comment:
We don't use wildcard imports, set the following options please: 
![image](https://user-images.githubusercontent.com/26204457/62480174-0acae000-b7a7-11e9-8963-89b4672f494a.png)

Reply:
Done